TMC5130-EVAL/TMC5130_TMC_API/TMC5130_Simple_Rotation.c

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+* Copyright © 2024 Analog Devices Inc. All Rights Reserved.
+* This software is proprietary to Analog Devices, Inc. and its licensors.
+*******************************************************************************/
+
+#include "TMC5130_Simple_Rotation.h"
+
+/*
+ * Configures the registers with the right settings that are needed for rotating the motor.
+ * E.g Enabling driver, setting IRUN current etc.
+ */
+void initAllMotors(uint16_t icID)
+{
+    tmc5130_writeRegister(icID, TMC5130_GCONF, 0x00000000);  // Digital current scaling, SpreadCycle mode
+    
+    // IHOLD_IRUN: Set max current (IRUN = 31), and hold current (~30% of IRUN)
+    tmc5130_writeRegister(icID, TMC5130_IHOLD_IRUN, 0x0000AF08); // Max run current, 30% hold current
+    
+    // CHOPCONF: VSENSE = 1, microstepping 1/16, TOFF = 3
+    tmc5130_writeRegister(icID, TMC5130_CHOPCONF, 0x000101D5);
+    
+    // PWMCONF: Enable StealthChop, autoscale PWM, amplitude ~ moderate
+    tmc5130_writeRegister(icID, TMC5130_PWMCONF, 0x000500C8);
+    
+    // Set Ramp mode to velocity mode (constant velocity rotation)
+    tmc5130_writeRegister(icID, TMC5130_RAMPMODE, 0x00000001); // Velocity mode
```

Comment:
Please make a new function 'tmc5130_rotateMotor' in TMC5130.c and set this register inside that function. Take TMC5272 as reference.